packages/web/src/common/store/pages/profile/lineups/tracks/retrieveUserTracks.ts

```diff
@@ -25,7 +24,7 @@ export function* retrieveUserTracks({
   getUnlisted = false
 }: RetrieveUserTracksArgs): Generator<any, Track[], any> {
   const apiClient = yield* getContext('apiClient')
-  const apiTracks = yield* call(apiClient.getUserTracksByHandle, {
+  const apiTracks = yield apiClient.getUserTracksByHandle({
```

Comment:
yield* not needed here?

Reply:
testing

Reply:
yeah yield* gives a type error, but yield doesn't